### PR TITLE
https://issues.redhat.com/browse/ACM-5306

### PR DESCRIPTION
**Pull request by oafischer**
For 2.8 and 2.7.

We had this as a known issue previously so I added it there again. Let me know if we should move it somewhere else.

### CODE REVIEW

**Review by nyechiel — APPROVED** [review]

File: release_notes/known_issues_network.adoc [hunk 1]
```diff
@@ -76,5 +76,10 @@ spec:
    insecureBrokerConnection: true
 ----
 
+[#submariner-sdn-cni]
+=== Submariner only supports OpenShift SDN or OVN Kubernetes
+
+Submariner only supports OpenShift clusters that use the OpenShift SDN or the OVN Kubernetes CNI network provider.
```
Comment (dockerymick) [inline release_notes/known_issues_network.adoc]: It wasn't clear to me what the acronyms were for before I googled. For the sake of the header length, the acronyms are appropriate
```suggestion
Submariner only supports OpenShift clusters that use the OpenShift SDN or the OVN-Kubernetes Container Network Interface (CNI) network provider.
```

**Review by dockerymick — APPROVED** [review]
Thanks for the updates! Made a suggestion

File: release_notes/known_issues_network.adoc (hunk 1 above)
Comment (nyechiel) [reply release_notes/known_issues_network.adoc]: https://docs.openshift.com/container-platform/4.12/release_notes/ocp-4-12-release-notes.html#ocp-4-12-ovn-kubernetes-default-network-plug-in

Let's use "OVN-Kubernetes" and "OpenShift SDN" like the OCP docs does. These are well-known terms within OpenShift.

Comment (dockerymick) [reply release_notes/known_issues_network.adoc]: Updated my suggestion based on @nyechiel feedback

**Review by dockerymick — APPROVED** [review]

**Review by nyechiel — APPROVED** [review]

**Review by sridhargaddam — APPROVED** [review]

**Pull request merged**